src/xGadgetron/cGadgetron/tests/CMakeLists.txt

```diff
@@ -87,7 +100,23 @@ add_dependencies(MR_TESTS_CPLUSPLUS GENERATE_SIMULATED_TESTDATA PROCESS_SIMULATE
 target_link_libraries(MR_TESTS_CPLUSPLUS PUBLIC MR_TESTS_CPP_AUXILIARY csirf cgadgetron)
 INSTALL(TARGETS MR_TESTS_CPLUSPLUS DESTINATION bin)
 
+
+set(MR_TESTDATA_PATH "${SHARE_DIR}/data/examples/MR")
+if(RERUN_ISMRMRD_SHEPP_LOGAN)
+  install(FILES  "${TESTDATA_OUTPUT_PATH}/${TESTDATA_OUTPUT_NAME_FULL}"
+                 "${TESTDATA_OUTPUT_PATH}/${TESTDATA_OUTPUT_NAME_GRAPPA}"
+          DESTINATION "${MR_TESTDATA_PATH}")
```

Comment:
do we really want to install these?

Reply:
Well, I don't know, I mean I can run the tests on the files no matter where they are, I figured I would install them into the shared directory, that's where the data from the submodule end up as well.